roles/k8s_best_practices_certsuite/templates/certsuite_config.yml.j2

```diff
@@ -50,6 +50,15 @@ servicesignorelist:
   - {{ item }}
 {% endfor %}
 
+{% if kbpc_cwe_apikey is defined and kbpc_cwe_project_id is defined %}
```

Comment:
This should only be included if certsuite version is higher or equal than v5.4.2 or if using HEAD